package.json

```diff
@@ -35,46 +38,43 @@
   },
   "license": "Apache-2.0",
   "dependencies": {
-    "jwk-to-pem": "^2.0.0"
+    "jwk-to-pem": "^2.0.5",
+    "keycloak-admin-client": "bucharest-gold/keycloak-admin-client"
```

Comment:
Why do you need to move this dependency from dev ?

Reply:
I can't remember, but it may be that I spotted it in the product/prod-arguments.json and as such thought that this was used in the Redhat release process (I cannot find any docs or have seen any on what the release scripts and files are used for/how BTW).